app/experimenter/jetstream/tests/test_tasks.py

```diff
@@ -837,15 +838,62 @@ def test_data_fetch_in_loop(self, lifecycle, mock_delay):
         tasks.fetch_jetstream_data()
         mock_delay.assert_called_once_with(experiment.id)
 
+    @parameterized.expand(
+        [
+            (NimbusExperimentFactory.Lifecycles.LIVE_APPROVE_APPROVE,),
+        ]
+    )
```

Comment:
Why parameterize for a single option?

Reply:
Yea I just kept the same format as all the other existing tests, but you're right no reason to keep this pattern when it's a single option. Removed in latest commit.